app/schemas/models/concept.schema.js

```diff
@@ -0,0 +1,75 @@
+const schema = require('./../schemas')
+
+const ConceptSchema = schema.object(  {
+  description:
+    'A Computer Science Concepts like while loops, or nested if statements.',
+  title: 'Concept',
+  required: ['name']
+})
+_.extend(ConceptSchema.properties,{
+    levelsCount: {
```

Comment:
how will level and course count be calculated? Is someone entering manually or in some dynamic way? 

Do we have a single collection between oz and coco or is it different? Do we need product-specific count?

Reply:
we have one single collection. I'm not sure if we'd like to update it always. I think, it's just a number that roughly tells how frequently used the concept.